examples/export-docs/export-docs.cpp

```diff
@@ -0,0 +1,55 @@
+#include "common.h"
+#include "llama.h"
+
+#include <cmath>
+#include <cstdio>
+#include <fstream>
+#include <string>
+#include <vector>
+
```

Comment:
Remove the unused headers

Reply:
Done in https://github.com/ggerganov/llama.cpp/pull/9308/commits/4b96c69a08a0e0e6f09883f7d8bad1375fbcaf86